lms/templates/courseware/course_navigation.html

```diff
@@ -14,10 +14,11 @@
     # If active_page is not passed in as an argument, it may be in the context as active_page_context
     active_page = active_page_context
 
-include_special_exams = settings.FEATURES.get('ENABLE_SPECIAL_EXAMS', False) and (course.enable_proctored_exams or course.enable_timed_exams)
+if course is not None:
+    include_special_exams = settings.FEATURES.get('ENABLE_SPECIAL_EXAMS', False) and (course.enable_proctored_exams or course.enable_timed_exams)
 %>
 
-% if include_special_exams:
+% if include_special_exams is not UNDEFINED:
```

Comment:
What if `include_special_exams` is not undefined but `False`? We could do `include_special_exams is not UNDEFINED and include_special_exams`